cloud-formation/dev-template.json

```diff
@@ -155,6 +155,27 @@
                 "Bucket": { "Ref": "ThumbBucket" }
             }
         },
+        "OptimisedPngBucket": {
```

Comment:
@akash1810 I thought it would be cleaner to have the optimised pngs in a separate bucket because they exist fora particular purpose.